### ISSUE TITLE
Path error when running code.

### ISSUE THREAD

**Issue by YwEwanHuang** [spyder-ide/spyder#5422]
## Description

**What steps will reproduce the problem?**

1.  Running code would pop this window out
2.  Tried to reset spyder but nothing changed
3. 

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

  File "D:\engpath\Anaconda\envs\PY2\lib\site-packages\spyder\plugins\ipythonconsole.py", line 895, in set_working_directory
    refresh_console=False)
  File "D:\engpath\Anaconda\envs\PY2\lib\site-packages\spyder\plugins\workingdirectory.py", line 286, in chdir
    os.chdir( to_text_string(directory) )
WindowsError: [Error 3] : u'D:\\\ufffd\ufffd\ufffd\ufffd\\program\\Python\\Crawl 51go'

## Version and main components

* Spyder Version: 3.2.0 
* Python Version: 2.7.13
* Qt Versions: 5.6.2, PyQt5 5.6 on Windows

## Dependencies
```
pyflakes >=0.5.0  :  1.5.0 (OK)
pycodestyle >=2.3 :  2.3.1 (OK)
pygments >=2.0    :  2.2.0 (OK)
pandas >=0.13.1   :  0.20.3 (OK)
numpy >=1.7       :  1.13.1 (OK)
sphinx >=0.6.6    :  1.6.2 (OK)
rope >=0.9.4      :  0.9.4 (OK)
jedi >=0.9.0      :  0.10.2 (OK)
nbconvert >=4.0   :  5.2.1 (OK)
sympy >=0.7.3     :  None (NOK)
cython >=0.21     :  None (NOK)
qtconsole >=4.2.0 :  4.3.0 (OK)
IPython >=4.0;<6.0:  5.3.0 (OK)
pylint >=0.25     :  1.6.4 (OK)

```

**Comment by YwEwanHuang** [comment]
down spyder to version 3.1.2 solved this problem ... 
but still be appreciated if anyone can help me out under 3.2.0

**Comment by ccordoba12** [comment]
This error was fixed in Spyder 3.2.1. Please update.
